f8a_report/graph_report_generator.py

```diff
@@ -0,0 +1,221 @@
+"""Helper functions related to to generate ingestion reports."""
+
+from f8a_utils.versions import get_latest_versions_for_ep
+import logging
+import os
+import requests
+import traceback
+from requests.adapters import HTTPAdapter
+from requests.packages.urllib3.util.retry import Retry
+
+_logger = logging.getLogger(__name__)
+GREMLIN_SERVER_URL_REST = "http://gremlin-yzainee-fabric8-analytics.devtools-dev.ext.devshift.net"
+"""
+GREMLIN_SERVER_URL_REST = "http://{host}:{port}".format(
+    host=os.environ.get("BAYESIAN_GREMLIN_HTTP_SERVICE_HOST", "localhost"),
+    port=os.environ.get("BAYESIAN_GREMLIN_HTTP_SERVICE_PORT", "8182")) """
+
+GREMLIN_QUERY_SIZE = os.getenv('GREMLIN_QUERY_SIZE', 50)
+
+"""
+TODO: WIP
```

Comment:
Let us not push any WIP code to the master even if those are commented out :-)

Reply:
removed